framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -2327,4 +2232,43 @@ public AnnotatedTypeMirror getDummyAssignedTo(ExpressionTree expressionTree) {
         }
         return null;
     }
+
+    /**
+     * Unboxes a type it needed all the way to a primitive integral.
+     *
+     * @param type TypeMirror to unbox.
```

Comment:
Minor:  In a Javadoc `@param`, `@return`, etc. clause, the initial text is a sentence fragment that starts with a lowercase (not capital) letter (which you did, thanks) and does not end with a period unless followed by another sentence.

I don't make the rules; those are just the conventions.

Reply:
OK